### PR TITLE
Manually add in [] stripped by elasticsearch_dsl, remove vestigial file

### PR DESCRIPTION
**Pull request by ChefAndy**
* modified: capapi/documents.py
	* to_dict
		* overrode to add skip_empty=False, supported by new elasticsearch_dsl
		* recreated structure of returned volume to include empty lists... skip_empty not yet applied to embedded objects
* deleted: capapi/views/doc_views.py
	* I think this is vestigial
* modified: docker-compose.yml requirements.in requirements.txt
	* specified updated versions
		* elasticsearch_dsl
		* django-elasticsearch_dsl
		* elasticsearch_dsl_drf

### CODE REVIEW

File: capstone/requirements.in [hunk 1]
```diff
@@ -89,6 +89,6 @@ mincss       # remove unused css styles
 requests  # also via aws-xray-sdk, coreapi, docker, moto, response
 
 # Elasticsearch
-elasticsearch-dsl
-django-elasticsearch-dsl
-django-elasticsearch-dsl-drf
+elasticsearch-dsl==6.4.0
```
Comment (jcushman) [inline capstone/requirements.in]: We don't want to specify versions in requirements.in -- that's what requirements.txt is for.

The simplest way to upgrade packages is with pip-compile's `-P` flag -- in this case, that would be `fab pip-compile:"-P elasticsearch-dsl django-elasticsearch-dsl django-elasticsearch-dsl-drf"`. At this point you could also just roll back the changes to requirements.in, which has the same effect.

File: capstone/docker-compose.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ services:
           - 9200:9200
     worker:
         build: .
-        image: capstone:0.3.59-b5782fc50cfbbaa3c159988ca2e8abf1
+        image: capstone:0.3.60-1f51a4b03f405f7e3ef01a680aafaca1
```
Comment (jcushman) [inline capstone/docker-compose.yml]: Oops, this has to be updated because of a parallel PR from Ben.

**Review by jcushman — CHANGES_REQUESTED** [review]
Thanks! Just a couple of tweaks.

File: capstone/docker-compose.yml (hunk 2 above)
Comment (ChefAndy) [reply capstone/docker-compose.yml]: Donesky

File: capstone/requirements.in (hunk 1 above)
Comment (ChefAndy) [reply capstone/requirements.in]: Done

**Pull request merged**